fdbserver/include/fdbserver/IEncryptionKeyProvider.actor.h

```diff
@@ -238,8 +240,8 @@ class TenantAwareEncryptionKeyProvider : public IEncryptionKeyProvider {
 	                                            const KeyRef& end,
 	                                            EncryptCipherDomainNameRef* domainName) {
 		int64_t domainId = SYSTEM_KEYSPACE_ENCRYPT_DOMAIN_ID;
-		int64_t beginTenantId = getTenant(begin, true /*inclusive*/);
-		int64_t endTenantId = getTenant(end, false /*inclusive*/);
+		int64_t beginTenantId = getTenantId(begin, true /*inclusive*/);
+		int64_t endTenantId = getTenantId(end, false /*inclusive*/);
```

Comment:
This one can also use the boolean params mechanism

Reply:
I thought about it, but then would let @sfc-gh-yiwu to make necessary changes as he is still working on Redwood Encyrption patch.